### PR TITLE
Update entry count in home page #7222

### LINKED ISSUE #7222: Update entry count in home page
Update this number to reflect 6.5 million as at May 4th 2017

image

### PR DESCRIPTION
**Pull request by shubham49**
Fixes #7222 
Updated index.es6 file.

### CODE REVIEW

**Comment by shubham49** [comment]
Ready for review.

File: src/main/webapp/js/index.es6 [hunk 1]
```diff
@@ -56,8 +56,8 @@ function loopTestimonials() {
 // Setting submission count at page load
 $('document').ready(() => {
     // Parameters for the estimation calculation
-    const baseDate = new Date('December 31, 2016 00:00:00');  // The date the parameters were adjusted
-    const baseCount = 5000000;     // The submission count on the above date
+    const baseDate = new Date('May 04, 2016 00:00:00');  // The date the parameters were adjusted
```
Comment (wkurniawan07) [inline src/main/webapp/js/index.es6]: Change the year to 2017 as well.

Comment (thenaesh) [inline src/main/webapp/js/index.es6]: My bad... I forgot it was 2017 too. xD

**Comment by shubham49** [comment]
Made the changes.

**Pull request merged**